EIPS/eip-draft_estimate_gas_return_data.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: _
+title: Estimate gas return data
+author: Jules Goullée (@julesGoullee)
+discussions-to: _
```

Comment:
You'll need a discussion-to link to get this merged.  The two most common places to do this are either a GitHub issue  or a thread in Ethereum Magicians forums.